### PR TITLE
[FIX] Add an extra column "tx_snowbabel_override_language_key" to avo…

### PR DESCRIPTION
**Pull request by GuillaumeGBzh**
…id conflicts between some languages #24

### CODE REVIEW

File: class.ext_update.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?php
```
Comment (WebsiteDeveloper) [inline class.ext_update.php:1]: Is there a reason as to why you did not use an upgrade wizard?
https://docs.typo3.org/typo3cms/CoreApiReference/ApiOverview/UpdateWizards/Index.html
Use of ext_update is discouraged.

File: ext_emconf.php [hunk 2]
```diff
@@ -17,12 +17,12 @@
 		'conflicts' => array(),
 		'suggests' => array(),
 	),
-	'state' => 'beta',
+	'state' => 'stable',
```
Comment (WebsiteDeveloper) [inline ext_emconf.php]: State of a new version is in beta until a release is tagged

File: ext_tables.sql [hunk 3]
```diff
@@ -14,6 +14,13 @@ CREATE TABLE be_users (
 	tx_snowbabel_languages TEXT
 );
 
+#
+# Add field to table 'static_languages'
+#
+CREATE TABLE static_languages (
+	tx_snowbabel_override_language_key varchar(5) DEFAULT NULL
```
Comment (WebsiteDeveloper) [inline ext_tables.sql:21]: The field should probably be added to tca

**Review by WebsiteDeveloper — COMMENTED** [review]
Mostly small notes.
Looks good in general.

File: ext_emconf.php (hunk 2 above)
Comment (WebsiteDeveloper) [reply ext_emconf.php]: On another note the version will be raised in a different commit so that change should probably be reverted

**Comment by WebsiteDeveloper** [comment]
Thank you for the PR 🎉

File: ext_tables.sql (hunk 3 above)
Comment (GuillaumeGBzh) [reply ext_tables.sql:21]: Done

File: class.ext_update.php (hunk 1 above)
Comment (GuillaumeGBzh) [reply class.ext_update.php:1]: TYPO3\CMS\Install\Updates\UpgradeWizardInterface isn't present in TYPO3 8 LTS. 
I have no other solutions.

Comment (WebsiteDeveloper) [reply class.ext_update.php:1]: Yes i forgot to add that the class has to extend AbstractUpdate see https://github.com/georgringer/news/blob/master/Classes/Updates/NewsSlugUpdater.php for example.

And the update Wizard can be added in localconf via:
    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update'][\GeorgRinger\News\Updates\NewsSlugUpdater::class]
        = \GeorgRinger\News\Updates\NewsSlugUpdater::class;

**Comment by WebsiteDeveloper** [comment]
@GuillaumeGBzh One last comment and then it looks good to go

**Pull request merged**